clients/cobol-lsp-vscode-extension/src/__tests__/commands/SmartTabCommandTest.spec.ts

```diff
@@ -28,19 +30,19 @@ const context: any = {
     subscriptions: [],
 };
 
-jest.mock("vscode", () => ({
-    window: {
-        activeTextEditor: {
-            options: {
-                tabSize: 4
-            }
-        }
-    },
-    commands: {
-        executeCommand: jest.fn().mockImplementation(),
-        registerTextEditorCommand : jest.fn().mockImplementation((command, callback) => callback(editor, {}))
-    },
-}));
+// jest.mock("vscode", () => ({
```

Comment:
Commented code should be removes or there should be a comment about this code - why it is commented